### DISCUSSION TITLE
Milvus cluster deployment with rocksdb

### DISCUSSION THREAD

**Discussion by phucng** [milvus-io/milvus#26890] [Q&A and General discussion] [answered]
I'm trying to reduce the footprint of the Milvus cluster deployment. Does it make sense if I use rocksdb instead of Kafka or Pulsar? If it does, how do we tell Milvus to use rocksdb in a cluster deployment? In the helm chart, the message queue parameter is only available for standalone deployments.

**Comment by xiaofan-luan** [comment]
rocksmq only valid for standalone option

**Comment by phucng** [comment]
> rocksmq only valid for standalone option

Do you mean rocksdb has some inherent limits, and therefore it is not supported in cluster deployments?

**Reply by xiaofan-luan** [reply, marked as answer]
yep, rocksdb is only for standalone

**Comment by mybabyiscue** [comment]
Hello, author. I saw the question you raised, so I guess you have already completed the cluster setup. I'm a newbie. Due to limited abilities. It is impossible to build a milvus cluster alone, so if you have relevant installation tutorials, can you provide one to me? If it's convenient for you, that would be very grateful.
